src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -488,14 +506,20 @@ private void getNextBatch(String requestId, Request request, boolean firstBatch,
       try {
         for (JsonEvent jsonEvent : batch) {
           JsonObject value = jsonEvent.objectValue();
-          String instanceId = value.getString(ENRICHED_INSTANCE_ID);
-          Object itemsandholdingsfields = value.getValue(RecordMetadataManager.ITEMS_AND_HOLDINGS_FIELDS);
+          String instanceId = value.getString(INSTANCE_ID_FIELD_NAME);
+          Object itemsandholdingsfields = value;
           if (itemsandholdingsfields instanceof JsonObject) {
```

Comment:
itemsandholdingsfields equals to variable value. Why do we need to have two the same variables?

Reply:
You are right, the redundant variable has been removed.